src/components/RightSidebar/RightSidebar.vue

```diff
@@ -268,7 +268,9 @@ export default {
 		},
 
 		showBreakoutRoomsTab() {
-			return this.getUserId && !this.isOneToOne && this.breakoutRoomsConfigured
+			return this.getUserId
+				&& !this.isOneToOne
+				&& (this.breakoutRoomsConfigured || this.conversation.breakoutRoomMode === CONVERSATION.BREAKOUT_ROOM_MODE.FREE || this.conversation.objectType === 'room')
```

Comment:
breakoutRoomsConfigured is always true when `.conversation.breakoutRoomMode === CONVERSATION.BREAKOUT_ROOM_MODE.FREE` is?

So
```suggestion
				&& (this.breakoutRoomsConfigured || this.conversation.objectType === 'room')
```

Should be good enough? Or am I missing something?